x-pack/plugins/security_solution/cypress/test_plans/detection_response/prebuilt_rules/original_scenarios.md

```diff
@@ -0,0 +1,264 @@
+
+```Gherkin
+Given user already has 8.x prebuilt rules installed
+And there are new prebuilt rules available to install
+When user navigates to the Rules Management Page
+Then user should see a callout message to install new prebuilt rules
+And the number of new rules available to install should be displayed on the +Add Elastic Rules link
+```
+
+#### **Scenario: Callout message is displayed when there are new updates on already installed prebuilt rules**
+
+```Gherkin
+Given user already has 8.x prebuilt rules installed
+And there are new updates available for those prebuilt rules
+When users navigate to the Rules Management Page
+Then users should see an update callout message
+And the number of outdated rules should be displayed on the Rules Updates tab
+```
+
+#### **Scenario: User is notified of available prebuilt rules to install when a rule is deleted**
+
+```Gherkin
+Given user has the latest version of prebuilt rules installed
+When user navigates to Rules Management Page
+And user deletes some prebuilt rules
+Then user should see a callout message with the same amount of prebuilt rules ready to install
+And the updated number of available rules to install should be displayed on the +Add Elastic Rules link
+```
+
+#### **Scenario: User is notified that all available rules have been installed in the Add Elastic Rules**
+
+```Gherkin
+Given user has all available rules installed
+When user navigates to the Add Elastic Rules Page
+Then user should see a message indicating that all available rules have been installed
+And user should see a CTA that leads to the Rules Management Page
+And user should navigate back to Rules Management Page when clicking on the CTA
+```
+
+#### **Scenario: User is notified that all installed rules are up to date in the Rule Updates tab**
+
+```Gherkin
+Given user has all available rules installed
+When user navigates to the Rule Update
+Then user should see a message indicating that all installed rules are up to date
+```
+
+### Prebuilt Rules Installation
+
+#### **Scenario: User without any installed prebuilt rules can install `<amount>` prebuilt rules**
+
+```Gherkin
+Given a user that doesn’t have prebuilt rules installed
+When user navigates to Add Elastic Rules Page
+Then available prebuilt rules are displayed on Elastic Rules table
+And user can install <amount> prebuilt Rules
+And successfully installed message is displayed after installation
+And installed rules are removed from Elastic Rules table
+And rules to install counter is decreased accordingly
+
+Examples:
+  | amount     |
+  | all        |
+  | selected   |
+  | individual |
+```
+
+#### **Scenario: User navigating to the Add Elastic Rules page sees a loading skeleton until the prebuilt rules package installation is completed**
+
+```Gherkin
+Given a user that is on Rules Management Page
+When user to the Add Elastic Rules page before rules package is installed
+Then a loading skeleton is displayed until the installation is completed
+```
```

Comment:
Replaced by:

#### **Scenario: User opening the Add Rules page sees a loading skeleton until the package installation is completed**

**Coverage**: ?.

```Gherkin
Given prebuilt rules package is not installed
When user opens the Add Rules page
Then user should see a loading skeleton until the package installation is completed
```